brontide/noise.go

```diff
@@ -744,7 +743,7 @@ func (b *Machine) WriteMessage(p []byte) error {
 // returned reflects the number of plaintext bytes in the payload, and does not
 // account for the overhead of the header or MACs.
 //
-// NOTE: It is safe to call this method again iff a timeout error is returned.
+// NOTE: It is safe to call this method again if a timeout error is returned.
```

Comment:
in this case, `iff` is correct as it means if-and-only-if

Reply:
gotya